backend/app/src/main/java/com/moh/phlat/backend/controller/ProcessDataController.java

```diff
@@ -348,57 +433,57 @@ public ResponseEntity<ResponseMessage> validateProcessDataById(@PathVariable Lon
 			
 		}
 		String authenticatedUserId= AuthenticationUtils.getAuthenticatedUserId();
-		dbUtilityService.setControlStatus(id, "PRE-VALIDATION_IN_PROGRESS",
+		dbUtilityService.setControlStatus(id, RowStatusService.PRE_VALIDATION_COMPLETED,
 										  authenticatedUserId);
 		// asynchronous operation
 		dbUtilityService.validateProcessDataByControlTableId(id,authenticatedUserId);
 
-		Optional<Control> _control = controlRepository.findById(id);
+		Optional<Control> control = controlRepository.findById(id);
 		
-		return ResponseEntity.status(HttpStatus.OK).body(new ResponseMessage("success", 200, "Validation process started!",_control));
+		return ResponseEntity.status(HttpStatus.OK).body(new ResponseMessage("success", 200, "Validation process started!", control));
 	}
 
 	@PreAuthorize("hasAnyRole(@roleService.getAllRoles())")
 	@PutMapping("/plrload/{controlTableId}")
 	public ResponseEntity<ResponseMessage> plrLoad(@PathVariable Long controlTableId) {
 
-		Optional<Control> _control = controlRepository.findById(controlTableId);
-		if (_control.isPresent()) {
-			Control control = _control.get();
-			if (!control.getStatusCode().equals("APPROVED")) {
+		Optional<Control> controlTable = controlRepository.findById(controlTableId);
+		if (controlTable.isPresent()) {
+			Control control = controlTable.get();
+			if (!control.getStatusCode().equals(RowStatusService.APPROVED)) {
 				return ResponseEntity.status(HttpStatus.NOT_FOUND).body(new ResponseMessage("error", 404, "Missing approval from Reg Admin to load to PLR.", "[]"));
 			}
 		}
 
-		List<ProcessData> _processDataList = processDataRepository.getAllProcessDataByControlTableId(controlTableId);
+		List<ProcessData> processDataList = processDataRepository.getAllProcessDataByControlTableId(controlTableId);
 		
-		if (_processDataList.isEmpty()) {
-			return ResponseEntity.status(HttpStatus.NOT_FOUND).body(new ResponseMessage("error", 404, "Nothing to load to PLR.", "[]"));
-			
+		if (processDataList.isEmpty()) {
+			return ResponseEntity.status(HttpStatus.NOT_FOUND).body(new ResponseMessage("error", 404, "Nothing to load to PLR.", "[]"));			
 		}
+		
 		String authenticatedUserId= AuthenticationUtils.getAuthenticatedUserId();
-		dbUtilityService.setControlStatus(controlTableId, "PLR_LOAD_IN_PROGRESS",
+		dbUtilityService.setControlStatus(controlTableId, RowStatusService.PLR_LOAD_IN_PROGRESS,
 										  authenticatedUserId);
 		// asynchronous operation
 		dbUtilityService.loadProcessDataToPlr(controlTableId,authenticatedUserId);
 		
-		return ResponseEntity.status(HttpStatus.OK).body(new ResponseMessage("success", 200, "PLR load process started!",_control));
+		return ResponseEntity.status(HttpStatus.OK).body(new ResponseMessage("success", 200, "PLR load process started!", controlTable));
 	}
 
 @PreAuthorize("hasAnyRole(@roleService.getAllRoles())")
 @GetMapping("/reportsummary/{controlTableId}")
 public @ResponseBody ResponseEntity<ResponseMessage> getReportSummaryByControlTableId(
```

Comment:
I feel the name is too generic. the name should be specific to business purpose

Reply:
I confirmed with our BA and she wants to call this as Report Summary.

Reply:
Ok. Since we did not provide complete context to BA, she will not understand why we are asking. Anyway, approving ...